site/src/components/accordion/Accordion.module.css

```diff
@@ -0,0 +1,94 @@
+.accordionContainer {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr) min-content;
+  grid-template-rows: repeat(3, min-content);
+  overflow: hidden;
+}
+
+.accordionIllustration {
+  grid-area: 1 / 2 / 3 / 4;
+}
+
+.wrapper {
+  background-color: var(--card-background);
+  width: 55vw;
+  padding: calc(var(--uitk-size-unit) * 10);
+  grid-area: 2 / 1 / 4 / 3;
+  display: flex;
+  flex-direction: column;
+  min-height: 70vh;
+}
+
+.accordion {
+  flex: 1;
+  padding: calc(var(--uitk-size-unit) * 2) 0;
+}
+
+.accordion .summary {
+  background-color: transparent;
+  padding-left: 0;
+}
+
+.accordion .summary:hover {
+  background: none;
+}
```

Comment:
This was a design requirement (having the highlight around the chevron only) but I'm happy for us to discuss with Shey

Reply:
Oh, ok. Yeah, let's discuss with Shey.

I think that's a bit misleading as it might create the false impression that only the chevron is clickable. Also, when the chevron rotates as the accordion opens/closes, the highlight does too which looks weird. :-P

Reply:
Agreed, it does look a bit funny!